examples/na_controlling_the_pml/example_na_controlling_the_pml.ipynb

```diff
@@ -24,8 +24,8 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# Install k-wave-python if not installed yet (e.g. running in Google Colab)\n",
-    "#!pip install git+https://github.com/waltsims/k-wave-python"
+    "%%capture\n",
+    "!pip install k-wave-python"
```

Comment:
What if we install only if in colab? We can detect running in colab using the snippet here: https://stackoverflow.com/a/53586419